core/commands/shutdown.go

```diff
@@ -3,31 +3,28 @@ package commands
 import (
 	"fmt"
 
+	cmds "gx/ipfs/QmNueRyPRQiV7PUEpnP4GgGLuK1rKQLaRW7sfPvUetYig1/go-ipfs-cmds"
 	"gx/ipfs/QmdE4gMduCKCGAcczM2F5ioYDfdeKuPix138wrES1YSr7f/go-ipfs-cmdkit"
-
-	cmds "github.com/ipfs/go-ipfs/commands"
 )
 
 var daemonShutdownCmd = &cmds.Command{
 	Helptext: cmdkit.HelpText{
 		Tagline: "Shut down the ipfs daemon",
```

Comment:
Maybe `Schedule shutdown of the ipfs daemon`?

Reply:
Well, it *does* shut it down. This will:

1. Cancel in-progress requests.
2. Wait for the IPFS node to close.
3. Return.

Note: it won't actually *wait* for all in-progress requests to finish. That would deadlock.